### PR TITLE
Add iAlarm integration

### PR DESCRIPTION
**Pull request by RyuzakiKK**
## Proposed change

Previous iAlarm integration has been removed because it used webscraping https://github.com/home-assistant/core/pull/43010 and https://github.com/home-assistant/home-assistant.io/pull/15584

Now pyialarm library has been updated to the version 1.0 that uses the iAlarm API instead https://github.com/RyuzakiKK/pyialarm/blob/master/pyialarm/pyialarm.py

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/43525
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/ialarm.markdown [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+This platform supports the following services: `alarm_arm_away`, `alarm_arm_home` and `alarm_disarm`.
+
+## Configuration
+
+To enable this, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+alarm_control_panel:
+  - platform: ialarm
```
Comment (frenck) [inline source/_integrations/ialarm.markdown]: We no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>
- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/ialarm.markdown [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+description: Instructions on how to integrate iAlarms alarms into Home Assistant.
+logo: antifurto365-ialarm.png
+ha_category:
+  - Alarm
+ha_iot_class: Local Polling
+ha_release: '2021.1'
+ha_domain: ialarm
+ha_codeowners:
+  - '@RyuzakiKK'
+ha_quality_scale: silver
```
Comment (RyuzakiKK) [inline source/_integrations/ialarm.markdown]: Not 100% sure about this one yet.

Comment (RyuzakiKK) [reply source/_integrations/ialarm.markdown]: Removed for now.

File: source/_integrations/ialarm.markdown [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+ha_config_flow: true
+ha_release: "2021.5"
+ha_domain: ialarm
+ha_codeowners:
+  - '@RyuzakiKK'
+---
+
+The `ialarm` platform provides connectivity with the [Antifurto365](https://www.antifurtocasa365.it/) iAlarm alarm systems and has also been confirmed to work with the alarm system brands Meian and Emooluxr.
+
+This platform supports the following services: `alarm_arm_away`, `alarm_arm_home` and `alarm_disarm`.
```
Comment (frenck) [inline source/_integrations/ialarm.markdown]:
```suggestion
The iAlarm integration provides connectivity with the [Antifurto365](https://www.antifurtocasa365.it/) iAlarm alarm systems and has also been confirmed to work with the alarm system brands Meian and Emooluxr.

This platform supports the following services:

- `alarm_control_panel.alarm_arm_away`
- `alarm_control_panel.alarm_arm_home`
- `alarm_control_panel.alarm_disarm`
```

**Review by frenck — APPROVED** [review]
Thanks, @RyuzakiKK 👍

**Pull request merged**